### DISCUSSION TITLE
docker-compose.yaml start failed

### DISCUSSION THREAD

**Discussion by cvpas** [dani-garcia/vaultwarden#3385] [Q&A] [answered]
Hello,I use docker-compose.yaml for all my docker projects.When I update my docker images to 1.28.0 from 1.27.0.
I find cannot start the vaultwarden container(repeat restarting).the docker-compose.yaml file as follows,If I delete all the "environment" parts,It can start normally.
Are there any changes for the environments setting?

```version: "3"

services:
  vaultwarden:
    image: vaultwarden/server:latest
    container_name: vaultwarden
    restart: always
    ports:
        - "127.0.0.1:8081:80"
        - "127.0.0.1:3012:3012"
    volumes:
      - ./vw-data:/data
    environment:
      WEBSOCKET_ENABLED: "true"
      SIGNUPS_ALLOWED: "false"
      WEB_VAULT_ENABLED: "true"
      SHOW_PASSWORD_HINT: "false"
      LOG_FILE: "/var/log/vaultwarden/vaultwarden.log"
      LOG_LEVEL: "warn"
      EXTEND_LOGGING: "true"
      ADMIN_TOKEN: "******...."
```

**Comment by BlackDex** [comment, marked as answer]
What are the errors of the docker start?
So, what do you see if you run either `docker logs vaultwarden` or `docker compose up` without the `-d`

**Reply by cvpas** [reply]
> What are the errors of the docker start? So, what do you see if you run either `docker logs vaultwarden` or `docker compose up` without the `-d`

<img width="652" alt="image" src="https://user-images.githubusercontent.com/36500518/228247453-9942d54a-1567-41b0-af9d-a39161ea8cc1.png">
the log file is exist in the folder

**Reply by BlackDex** [reply]
In which folder? On the host or the container? That variable needs to be the path within the container.
So you probably need to change it to `/data/vaultwarden.log`
That will make it appear in `./vw-data/vaultwarden.log` from your hosts perspective.

**Reply by cvpas** [reply]
Thanks! problem solved!

**Comment by bokkabonga** [comment]
Hard to tell without logs, but at least the variable "EXTEND_LOGGING" doesn´t exist. It should be "EXTENDED_LOGGING" as shown in

https://github.com/dani-garcia/vaultwarden/blob/main/.env.template

**Reply by BlackDex** [reply]
But, that doesn't prevent the container from starting. It just ignores that variable.
